src/sentry/utils/settings.py

```diff
@@ -70,3 +70,13 @@ def get_error_message(cls, dependency, package):
             """Python could not find %(package)s in your current environment (required by %(dependency)s). If you have it installed, maybe you are using the wrong python binary to run sentry?"""
             % {"dependency": dependency, "package": package}
         )
+
+
+def is_self_hosted():
+    # Backcompat for rename to support old consumers, particularly single-tenant.
+    from django.conf import settings
+
+    try:
+        return settings.SENTRY_SELF_HOSTED
+    except AttributeError:
+        return settings.SENTRY_ONPREMISE
```

Comment:
Nit: `return getattr(settings, 'SENTRY_SELF_HOSTED', 'SENTRY_ONPREMISE')`

Reply:
```
$ python3
Python 3.9.8 (main, Nov 10 2021, 06:03:50) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> class settings:
...   bar = 1
... 
>>> getattr(settings, 'foo', 'bar')
'bar'
>>>
```

Could do:

```
getattr(settings, 'foo', getattr(settings, 'bar'))
```

but is that really nicer? 🤔

Reply:
Oops yeah, I'm fine with it as is.